sdk/eventhub/azure-eventhubs/azure/eventhub/eventprocessor/sqlite3_partition_manager.py

```diff
@@ -34,17 +46,15 @@ def __init__(self, db_filename: str = ":memory:", ownership_table: str = "owners
         conn = sqlite3.connect(db_filename)
         c = conn.cursor()
         try:
-            c.execute("create table " + ownership_table +
-                      "(eventhub_name text,"
-                      "consumer_group_name text,"
-                      "owner_id text,"
-                      "partition_id text,"
-                      "owner_level integer,"
-                      "sequence_number integer,"
-                      "offset text,"
-                      "last_modified_time integer,"
-                      "etag text)")
+            sql = "create table if not exists " + _check_table_name(ownership_table)\
+                  + "("\
+                  + ",".join([x[0]+" "+x[1] for x in self.fields_dict.items()])\
+                  + ", constraint pk_ownership PRIMARY KEY ("\
+                  + ",".join(self.primary_keys)\
+                  + "))"
+            c.execute(sql)
         except sqlite3.OperationalError:
+            raise
             pass
```

Comment:
?

Reply:
Removed it. Sorry for pushing this code accidentally. I was thinking of raising a different type of exceptions but gave up.